### PR TITLE
feat(sponsors): add sponsors section

### LINKED ISSUE #420: Add sponsors
We're signing up the first sponsors that need to be added to the website. We'll need one pull request each for:

* redpencil at partner level – can reuse image and URL from previous years
* Qonto at partner level – can reuse image and URL from previous years
* Mainmatter as dinner sponsor – can reuse image and URL from previous years

### Resources

* Designs: https://www.sketch.com/s/150ed20a-d7c3-46a1-8151-89fba683acae/a/MmlKpPl

### PR DESCRIPTION
**Pull request by oscard0m**
# 📖 Description
- Add 'Sponsors' section
- remove 'Past Sponsors' section

## 🔬 Preview Link
🔗 https://deploy-preview-421--silly-leavitt-2e0668.netlify.app/

# 📚 Context
fixes #420

### CODE REVIEW

File: _includes/sponsors.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<section class="sponsors">
+	<h3>Sponsors</h3>
+  {% if site.data.sponsors.premier_partners %}
+  <div class="sponsor-section-wrapper">
+    <h4>Premier Partners</h4>
+    <ul class="sponsors-list premier-partner">
+      {% for sponsor in site.data.sponsors.premier_partners %}
+      <li>
+        <a href="{{ sponsor.link }}">
+          <img src="/2022/images/sponsors/{{sponsor.logo}}.svg" />
```
Comment (marcoow) [inline _includes/sponsors.html:10]:
```suggestion
          <img src="/images/sponsors/{{sponsor.logo}}.svg" />
```

let's not use images from 2022 in this year's page – that's confusing

**Review by marcoow — APPROVED** [review]
please don't merge yet though

**Comment by davidhajba** [comment]
I only see question marks instead of logos here in Safari, 
Do you have the same thing?

Screenshot 2023-07-24 at 09 35 01

**Comment by oscard0m** [comment]
Images are broken today but not last day, let me re-trigger Netlify

**Comment by oscard0m** [comment]
@davidhajba Ready to re-review

**Review by davidhajba — APPROVED** [review]

**Review by davidhajba — APPROVED** [review]

**Pull request merged**